wolfssl/wolfcrypt/md5.h

```diff
@@ -114,6 +117,11 @@ WOLFSSL_API int  wc_Md5Copy(wc_Md5*, wc_Md5*);
 WOLFSSL_API void wc_Md5SizeSet(wc_Md5* md5, word32 len);
 #endif
 
+#if defined(WOLFSSL_HASH_FLAGS) || defined(WOLF_CRYPTO_CB)
+    WOLFSSL_API int wc_Md5SetFlags(wc_Md5* md5, word32 flags);
```

Comment:
Do these need to be public?  In what cases should they be called by the user and how?  How do we make sure they're called correctly in the future?

Reply:
@toddouska: Very good point. I’ll make these local (private)

Reply:
@toddouska : I made these hash flag API's private.